src/components/routes/RouteTabs.vue

```diff
@@ -0,0 +1,145 @@
+<script lang="ts">
+/**
+ * RouteTabs Component
+ *
+ * @description * Use this component to render tabs on the Routes page.
+ *
+ * @slots
+ * - `calendar`: For calendar view.
+ * - `list`: For list view.
+ * - `map`: For map view.
+ * - `app`: For a view of app links.
+ *
+ * @components
+ * - `RoutesCalendar`: Component to render the calendar view.
+ * - `RouteListDisplay`: Component to render the list view (display mode)
+ * - `RoutesListEdit`: Component to render the list view (edit mode)
+ *
+ * @example
+ * <route-tabs></route-tabs>
+ *
+ * @see [Figma Design](https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=4858%3A106366&mode=dev)
+ */
+
+// libraries
+import { defineComponent, ref } from 'vue';
+
+// components
+import RouteListDisplay from './RouteListDisplay.vue';
+
+// types
+import type { RouteItem, RouteTab } from '../types/Route';
+
+// fixtures
+import routeListFixture from '../../../test/cypress/fixtures/routeList.json';
+// @ts-expect-error to cast routeListFixture to RouteItem type
+const routeList: RouteItem[] = routeListFixture as RouteItem[];
+
+export default defineComponent({
+  name: 'RouteTabs',
+  components: {
+    RouteListDisplay,
+  },
+  props: {
+    locked: {
+      type: Array as () => RouteTab[],
+      default: () => [],
+    },
+  },
+  setup(props) {
+    const activeTab = ref('');
+    // list locked tabs - exposed for testing and further logic
+    const lockedTabs = props.locked;
+    // getter function for locked state
+    const isLocked = (tab: RouteTab): boolean => {
+      if (!lockedTabs.length) return false;
+      return lockedTabs.includes(tab);
+    };
+
+    return {
+      activeTab,
+      lockedTabs,
+      routeList,
+      isLocked,
+    };
+  },
+});
+</script>
+
+<template>
+  <div>
+    <!-- Tab buttons -->
+    <q-tabs
+      inline-label
+      v-model="activeTab"
+      class="text-grey"
+      active-color="primary"
+      indicator-color="primary"
+      align="center"
+      data-cy="route-tabs"
+    >
+      <q-route-tab
+        to="/routes/calendar"
```

Comment:
URL path should not be **hardcoded in code**, reuse URL path from [src/router/routes_conf.ts](https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/353/files#diff-d1e26baefcd37ed6f5e2b78ded321ca99c0b50423b21cc1ca3587fdbb7e513c7) file please.

Line 92, 102, 112.

Reply:
Yes. Fixed.